poetry.lock

```diff
@@ -3065,7 +3065,6 @@ version = ">=3.7.4"
 [[package]]
 category = "main"
 description = "Backport of pathlib-compatible object wrapper for zip files"
-marker = "python_version < \"3.7\" and python_version != \"3.4\" or python_version < \"3.8\""
```

Comment:
Not sure where this comes from... I only updated the section for `tensorflow-text` in the `pyproject.toml` and then ran `poetry update tensorflow-text`.